frontend/src/components/Shared/Footer.jsx

```
@@ -1,30 +1,56 @@
-      <nav>
-        <h6 className="footer-title">Legal</h6>
-        <a href="#" className="link link-hover">Terms of use</a>
-        <a href="#" className="link link-hover">Privacy policy</a>
-        <a href="#" className="link link-hover">Cookie policy</a>
-      </nav>
-    </footer>
-  );
+	return (
+		<footer className="footer footer-center bg-[#323d5e] text-white rounded p-10">
+			<nav className="grid grid-flow-col gap-4">
+				<a className="link link-hover">About us</a>
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)